pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -31,6 +31,7 @@
   private static final String CONTROLLER_PORT = "controller.port";
   private static final String DATA_DIR = "controller.data.dir";
   private static final String ZK_STR = "controller.zk.str";
+  private static final String UPDATE_SEGMENT_STATE_MODEL = "controller.update_segment_state_model"; // boolean: Update the statemodel on boot?
```

Comment:
Re-thinking, we should not be changing state models that often (actually, never :-), so if we ever do again (which should not be anytime in the near future), we should be fine re-using this config to do the exact same thing -- update state model in place. Also, upgrade_to_llc_state_model, may not cover an upgrade to a llc state model that includes (say) a PAUSED state. 
In summary,let us stick with the existing config.
thanks